content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Announcing Istio 1.9.0
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** `istioctl install` will detect different Istio version installed (istioctl, control plan version) and display warning.
+  ([Issue #18487](https://github.com/istio/istio/issues/18487))
+
+- **Added** post-install/in-place upgrade verification of control plane health. Use `--verify` flag with `istioctl install` or `istioctl upgrade`  ([Issue #21715](https://github.com/istio/istio/issues/21715))
+
+- **Added** Add pprof endpoint to pilot-agent.
+  ([Issue #28040](https://github.com/istio/istio/issues/28040))
+
+- **Added**  `enableIstioConfigCRDs` to `base` to allow user specify whether the istio crds will be installed. ([Pull Request #28346](https://github.com/istio/istio/pull/28346))
+
+- **Added** Allow enabling gRPC logging with --log_output_level for pilot.
+  ([Issue #28482](https://github.com/istio/istio/issues/28482))
+
+- **Added** `istioctl apply` as an alias for `istioctl install`.
+  ([Issue #28753](https://github.com/istio/istio/issues/28753))
+
+- **Added** `--browser` flag to `istioctl dashboard`, which controls whether you want to open a browser to view the dashboard.
+  ([Issue #29022](https://github.com/istio/istio/issues/29022))
+
+- **Added** `istioctl verify-install` will indicate errors in red and expected configuration in green.
+  ([Issue #29336](https://github.com/istio/istio/issues/29336))
+
+- **Added** the severity level for each analysis message in the `validationMessages` field within the `status` field.  ([Issue #29445](https://github.com/istio/istio/issues/29445))
+
+- **Added** `WorkloadEntry` resources will be read from all clusters in multi-cluster installations and do not need to be duplicated.
+Makes Virtual Machine auto-registration compatible with multi-primary multi-cluster. This feature is disabled by default and can be
+enaled by setting the `PILOT_ENABLE_CROSS_CLUSTER_WORKLOAD_ENTRY` environment variable in istiod.
+  ([Issue #29026](https://github.com/istio/istio/issues/29026))
+
+- **Added** `istioctl analyze` now informs if deprecated or alpha-level annotations are present.
+(These checks can be disabled using `--suppress "IST0135=*"` and `--suppress "IST0136=*"`
+respectively.)
+  ([Issue #29154](https://github.com/istio/istio/issues/29154))
+
+- **Added** option to enable STS token fetch and exchange for XDS flow.
+  ([Issue #29943](https://github.com/istio/istio/issues/29943))
```

Comment:
@lei-tang could this be elaborated on user impact?  Sorry I don't understand this.

Reply:
@shamsher31 Can you revise "**Added** option to enable STS token fetch and exchange for XDS flow." into "**Added** option to allow users to enable token exchange for their XDS flows, which exchanges a k8s token to a token that can be authenticated at their XDS servers."?

Reply:
Done. Thanks.